src/helpers/marketstackHelper.ts

```diff
@@ -144,3 +146,37 @@ export const persistEODDataForPastNYears = async (
 
   return persistedEodData;
 };
+
+export const doesSecurityExist = async (symbol: string): Promise<boolean> => {
```

Comment:
NBD, but mismatch in naming convention between the prisma schema and here, ticker used in prisma nad symbol used here, should use the same